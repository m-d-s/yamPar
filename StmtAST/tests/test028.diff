0a1,7
> Call:   Top
>   Call:   Stmt
>     Call:   While
>       Consumed token: <"while" at line 1 column 1>
>     Return: While
>   Return: Stmt
> Return: Top
