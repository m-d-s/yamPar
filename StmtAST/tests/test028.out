Call:   Top
  Call:   Stmt
    Call:   While
      Consumed token: <"while" at line 1 column 1>
    Return: While
  Return: Stmt
Return: Top
Invalid syntax at (1,1), while
